CMakeLists.txt

```diff
@@ -479,8 +479,8 @@ endif()
 if(OPENVDB_BUILD_PYTHON_MODULE OR (OPENVDB_BUILD_NANOVDB AND NANOVDB_BUILD_PYTHON_MODULE))
 
   # Call find_package(Python ...)
-  find_package(Python 3.8 REQUIRED COMPONENTS Development Interpreter)
-  find_package(nanobind REQUIRED)
+  find_package(Python ${MINIMUM_PYTHON_VERSION} REQUIRED COMPONENTS Development Interpreter)
+  find_package(nanobind ${MINIMUM_NANOBIND_VERSION} REQUIRED)
```

Comment:
Because nanobind does not store version number in `2.0.0` to be ingested by CMake, should we take the ${MINIMUM NANOBIND_VERSION` here?